src/main/java/org/myrobotlab/inmoov/Utils.java

```diff
@@ -12,6 +13,9 @@
 
   public final static Logger log = LoggerFactory.getLogger(Utils.class);
 
+  transient private static Runtime myRuntime = (Runtime) Runtime.getInstance();

```

Comment:
I think if you have this here, it will cause runtime to be instantiated any time this Utils class is loaded into the classpath.  I don't think we want this behavior.